### PR TITLE
[native]Improve task manager shutdown logging

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Improve task manager shutdown logging by printing out
the task ids with pending references.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by pranjalssh — APPROVED** [review]

File: presto-native-execution/presto_cpp/main/common/Configs.cpp [hunk 1]
```diff
@@ -432,6 +433,10 @@ bool SystemConfig::enableMemoryArbitration() const {
   return optionalProperty<bool>(kEnableMemoryArbitration).value_or(false);
 }
 
+std::string SystemConfig::memoryArbitratorKind() const {
```
[the remark this answers is not in the document]
Comment (xiaoxmeng) [reply presto-native-execution/presto_cpp/main/common/Configs.cpp:436]: We need to do this in followup. Thanks!

File: presto-native-execution/presto_cpp/main/TaskManager.h [hunk 2]
```diff
@@ -32,6 +32,10 @@ class TaskManager {
  public:
   TaskManager();
 
+  /// Invoked by Presto server shutdown to wait for all the tasks to complete
+  /// and cleanup the completed tasks.
+  void stop();
```
Comment (tanjialiang) [inline presto-native-execution/presto_cpp/main/TaskManager.h:37]: Do we want to call it shutdown()?

File: presto-native-execution/presto_cpp/main/TaskManager.cpp [hunk 3]
```diff
@@ -1008,19 +997,38 @@ std::array<size_t, 5> TaskManager::getTaskNumbers(size_t& numTasks) const {
...
   size_t numTasks;
   auto taskNumbers = getTaskNumbers(numTasks);
   size_t seconds = 0;
   while (taskNumbers[velox::exec::TaskState::kRunning] > 0) {
-    LOG(INFO) << "Waiting (" << seconds
-              << " seconds so far) for 'Running' tasks to complete. "
-              << numTasks << " tasks left: "
-              << PrestoTask::taskNumbersToString(taskNumbers);
+    PRESTO_SHUTDOWN_LOG(INFO)
+        << "Waiting (" << seconds
```
Comment (tanjialiang) [inline presto-native-execution/presto_cpp/main/TaskManager.cpp:1006]: Waited ... seconds so far

**Comment by amitkdutta** [comment]
Looks like e2e test failure with update velox
https://app.circleci.com/pipelines/github/prestodb/presto/5978/workflows/1d8de133-dd5b-4ff6-a478-6ea25cbe435b/jobs/17298

```
VeloxRuntimeError: isValid String view at idx 1518 points outside of the string buffers
```

CC: @mbasmanova

**Review by amitkdutta — APPROVED** [review]

**Pull request merged**